lib/guardian/category_guardian.rb

```diff
@@ -46,6 +46,11 @@ def cannot_delete_category_reason(category)
     nil
   end
 
+  def can_see_serialized_category?(category_id:, read_restricted: true)
```

Comment:
Extra guard here by always defaulting to read_restricted true.

Reply:
Can the category[:read_restricted] on this line ever be nil or is it always true/false?

https://github.com/discourse/discourse/pull/13433/files#diff-a73755563e0f7c1dcbd6d42d4cd961ba70c6744fde3f3e8ef27c9b8d7a0e9cceR63

Reply:
It is a boolean column :)